contracts/dev-utils/contracts/src/OrderValidationUtils.sol

```diff
@@ -118,22 +118,14 @@ contract OrderValidationUtils is
             transferableTakerAssetAmount
         );
 
-        // Execute the maker transfers.
-        fillableTakerAssetAmount = LibOrderTransferSimulation.getSimulatedOrderMakerTransferResults(
```

Comment:
:raised_hands: 

No one else is using this library now right? Can we delete it?